pkg.list

```diff
@@ -26,6 +26,8 @@ editors/libreoffice
 editors/vim
 emulators/virtualbox-ose-additions
 french/aspell
+french/hunspell
+french/mythes
```

Comment:
If we add `french/mythes` we'd have to add all the other \*-mythes, too, to be fair. This would, however, take 100 MB.

Reply:
I understood that can take more space on a USB stick, but I think if we will translate, we have to make it completely, no?

Maybe we can install theses packages during installation process on a hdd? Just after selecting region and language, maybe we can select the appropriate packages to installing? It's a part of translate the distribution.

Reply:
Sorry for the delay. The question is: are the \*-mythes packages really needed for creating translations? Writing translations doesn't even require NomadBSD. And if you need it you can install one or all of them via pkg, but most user will probably never use \*-mythes. The only ports I see that use textproc/mythes are:

- editors/openoffice-devel
- editors/libreoffice
- editors/openoffice-4
- print/lyx

Reply:
Sure, mythes packages are not essential, but is just on the continuity to translate the system and the applications. Like an Ubuntu. When you install an Ubuntu desktop, choose your region and language then after the installation all the system and applications are translated. It's rare to add some translation packages manually.

But, if this is not the vision of NomadBSD, that's fine for me, I'd just identified this need when I installing `devel/codeblocks`, which advertise you on the settings page that the thesaurus file was not found.

I close the PR.